backend/models/meta.js

```diff
@@ -241,7 +241,7 @@ class Meta {
 		models.add(model);
 
 		// Check submodels
-		const refs = await Ref.find({account, model}, {type: "ref"});
+		const refs = await findRef(account, model, {type: "ref"});
```

Comment:
A lot of these queries are not checking current IDs, considering we're passing in branch/revId it's probably worth considering if we actually have a bug here.... 😆 If it's a federation with revisions where we removed/added some sub models, are we still searching through some sub models that are no longer a submodel?